cmd/ipfs-cluster-ctl/main.go

```diff
@@ -914,7 +914,10 @@ but usually are:
 					Action: func(c *cli.Context) error {
 						metric := c.Args().First()
 						if metric == "" {
-							checkErr("", errors.New("provide a metric name"))
+							fmt.Printf("Try with one of the metrics names listed below:\n\n")
```

Comment:
avoid random text getting into the output. This does not make a text interface easy to work with when sedding, grepping, etc.

Instead, update the documentation of the command which is outdated.